apps/meteor/app/lib/server/lib/loginErrorMessageOverride.ts

```diff
@@ -0,0 +1,16 @@
+// Do not disclose if user exists when password is invalid
+import { Accounts } from 'meteor/accounts-base';
+import { Meteor } from 'meteor/meteor';
+
+const { _runLoginHandlers } = Accounts;
+
+Accounts._options.ambiguousErrorMessages = true;
+Accounts._runLoginHandlers = async function (methodInvocation, options) {
+	const result = await _runLoginHandlers.call(Accounts, methodInvocation, options);
+
+	if (result.error instanceof Meteor.Error) {
```

Comment:
All errors here should be considered as a User not found? Asking because we do have some examples like this one: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/apple/server/loginHandler.ts#L36-L38
returning other things as errors.

Also what about regular `Error` (not Meteor ones), should this be considered a problem?

Reply:
I was expecting at least ONE test telling me that we use some error for a different case.

Reply:
since now we are considering this as a break change 🤷🏻‍♂️ no need to worry about